js/tests/visual/collapse.html

```diff
@@ -57,7 +57,7 @@ <h5 class="mb-0">
       </div>
     </div>
 
-    <script src="../../../docs/assets/js/vendor/jquery-slim.min.js"></script>
```

Comment:
Can you go through all visual test files and see if there are more references like that and replace all of them?

Reply:
It was the only one I forget 😄